ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1860,6 +1860,8 @@ public string GetServerArgs()
                 serverArgs.Append($"?GameModIds={this.ServerModIds}");
             }
 
+            if (!String.IsNullOrWhiteSpace(this.AdditionalArgs) && !this.AdditionalArgs.StartsWith(" "))
```

Comment:
It is unlikely additional args would ever start with a space.  Because it is also being added unconditionally even if empty, I would simplify this as:
serverArgs.Append(' ').Append(this.AdditionalArgs);

The potential extra space on the command line will be ignored.